packages/async-operations/package.json

```diff
@@ -30,7 +30,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@aws-sdk/client-ecs": "^3.477.0",
-    "@aws-sdk/client-lambda": "^3.447.0",
+    "@aws-sdk/client-lambda": "^3.529.1",
```

Comment:
For the initial passes we were using the same 3.477 version but a decent amount of time has passed. It might be worth making it an AC on these tickets to go in and update the package you're working on to the most recent.